diff --git a/field_collection.entity.inc b/field_collection.entity.inc
index 2e8a11b..fb90ed2 100644
--- a/field_collection.entity.inc
+++ b/field_collection.entity.inc
@@ -523,6 +523,10 @@ class FieldCollectionItemEntity extends Entity {
     return drupal_map_assoc(array_keys($vars));
   }
 
+  static function __set_state(array $array) {
+    return $array;
+  }
+
   /**
    * Magic method to invoke setUp() on unserialization.
    *
